src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle implements Figure {
+    private int radius;
+
+    public Circle(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public String drawFigure() {
+        return "Shape: circle, area: " + getSquare() + ", radius: "
+                + radius + ", color: " + getColor();
+    }
+
+    @Override
+    public double getSquare() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public Color getColor() {
+        return new ColorProducer().insertRandomColor();
```

Comment:
your figure has a color (it should be a field) just return its value in the method getColor()